cmd/native-relayer/cli/utils.go

```diff
@@ -0,0 +1,39 @@
+package cli
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/spf13/viper"
+)
+
+func loadConfig() (*Config, error) {
+	var configFile string
+	flag.StringVar(&configFile, "config", "", "Path to the config file")
+	flag.Parse()
+
+	viper.SetConfigName("config")
+	viper.SetConfigType("yaml")
+	viper.AddConfigPath(".")
+	viper.AddConfigPath("$HOME/.native-relayer")
```

Comment:
Path should be input by user

Reply:
if the user provides the config file using `-config` flag then we will overwrtie it